### ISSUE TITLE
Better UX for bad networks

### ISSUE THREAD

**Issue by IpsmLorem** [wireapp/wire-ios#311]
When you send a huge picture and then a message over a very bad network, the user experience could be improved.
The picture sending lack of propoer ETA/visual progression of upload to the user. "Sending" doesn't provide enough information to user (Do I wait for nothing ? Is there something uploading or the network is completely stalled ?)
While the picture is sending, the message fail to be sent (and it doesn't automaticaly retry as stated in #197)
When the picture is sent, the failed message came on top, even if the creation timestamp is later. As soon as the retry was taped and sucessfuly sent, the message came back at te bottom, where it belong.

To sum up:
* sending media should display a progression of upload
* item display should follow the creation timestamp order, even in case of failure

**Comment by david-wire** [comment]
Hello, thanks for the feedback and for giving us specific details regarding this area, we'll take into consideration.
